lms/static/scripts/frontend_apps/components/JSTORPicker.js

```diff
@@ -72,17 +72,21 @@ export default function JSTORPicker({ onCancel, onSelectURL }) {
       metadata.error,
       'Unable to fetch article details'
     );
-  } else if (metadata.data?.is_collection) {
+  } else if (metadata.data?.content_status === 'no_content') {
     renderedError =
-      'This work is a collection. Enter the link for a specific article in the collection.';
+      'No content is available for this item. For collections, enter the link for a specific article or chapter.';
```

Comment:
How about:

"There is no content available for this item. To select an item within a journal or book, enter a link to a specific article or chapter."

I think I'm just getting hung up on "collections".

Reply:
This proposal sounds good. I've changed the text accordingly.